modules/shared/server/controllers/auth.helpers.js

```diff
@@ -8,6 +8,8 @@ const argon2 = require('argon2');
 // unique string generator
 const uuid = require('uuid');
 
+const config = require('../../../../config/config');
```

Comment:
NO ACTION NEEDED:

I feel like we should move the config file so that the module loader injects it to all modules...  I always feel like the far off require's are a bad way to do this.